.circleci/config.yml

```diff
@@ -83,7 +83,6 @@ jobs:
               if [[ ${project} =~ "yarn" || ${project} =~ ".buildkite" || ${project} =~ ".circleci" ]]; then
                 buildall=1
                 echo -e "Warning. Dependencies or internal configs have change detected."
-                exit 1
```

Comment:
Hey @briemcnally! Could you confirm if `exit 1` was intended? Does this fix looks good to you?

Reply:
@fathyb, Carlos had originally wanted me to implement this change because we are using the analytics.js-integrations repo has been migrated to Buildkite to use to run our own CI internally (for Segmenters). The Destinations team has a lot of external contributions from contractors, so in that push, we wanted to be able to keep the circle/config when we migrated to Buildkite to test PRs from external contributors. We would want to keep this here so it would fail if the config is changed by external contractors? 

It looks like the buildkite config is failing because of this: https://github.com/segmentio/analytics.js-integrations/blob/aa10d2c750456e3a57a642b92a861fda4a913387/.buildkite/pipeline.yml#L10. Which points to this: https://github.com/segmentio/analytics.js-integrations/blob/aa10d2c750456e3a57a642b92a861fda4a913387/package.json#L15

Reply:
Oh, that makes a lot of sense. Thanks for confirming!